fs/devfs/DevFS.c

```
@@ -0,0 +1,56 @@
+#include "../CharDevice.h"
+#include "../../kernel/etc/Console.h"
+#include "../../kernel/etc/StringOps.h"
+
+int DevfsMount(struct BlockDevice* device, const char* mount_point) {
+    // DevFS is a virtual filesystem and doesn't need a device.
+    (void)device;
+    (void)mount_point;
+    return 0;
+}
+
+int DevfsReadFile(const char* path, void* buffer, uint32_t max_size) {
```

Clang (14.0.6)

[warning] parameter 'max_size' is unused

(misc-unused-parameters)